python/docs/_reference.yml

```diff
@@ -0,0 +1,200 @@
+quartodoc:
+  title: API
+  dir: reference
+  package: kaskada
+  source_dir: ../pysrc
+  options:
+    children: separate
+    include_empty: true
+  dynamic: true
+
+  sections:
+    - title: Sessions
+      options:
+        children: embedded
+      contents:
+        - init_session
+    - title: Timestream
+      options:
+        children: flat
+      contents:
+        - name: Timestream
+          members:
+            - data_type
+            - is_continuous
+        - name: LiteralValue
+          dynamic: false
+        - name: Arg
+          dynamic: false
+    - subtitle: Aggregation
+      desc: |
```

Comment:
Idea for cleaning this up -- what if we had a file for this, and made this:

```
- subtitle: Aggregation
  desc: {{< include _aggregation.qmd >}}
  contents:
    ...
```

Benefits:
1. Don't need to know to look in `_reference.yml` to find the description of each section.
2. Can use normal markdown editor since it's a QMD file rather than a string embedded in yaml.